src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    if (!(robot[key] in reversedRobot)) {
+      reversedRobot[robot[key]] = key;
+    } else {
+      return null;
+    }
```

Comment:
було б круто замінити на просто if
```suggestion
    if (robot[key] in reversedRobot) {
      return null;
    }
    
    reversedRobot[robot[key]] = key;
```